Index: src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>spring.application.name=demo\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/application.properties b/src/main/resources/application.properties
--- a/src/main/resources/application.properties	(revision da111c4c508e794a3384304c03360b5ec0d89148)
+++ b/src/main/resources/application.properties	(date 1772193083480)
@@ -1,1 +1,2 @@
+spring.application.name=maternal-care
 spring.application.name=demo
Index: src/main/java/com/maternalcare/demo/model/TestEntity.java
===================================================================
diff --git a/src/main/java/com/maternalcare/demo/model/TestEntity.java b/src/main/java/com/maternalcare/demo/model/TestEntity.java
new file mode 100644
--- /dev/null	(date 1772193083517)
+++ b/src/main/java/com/maternalcare/demo/model/TestEntity.java	(date 1772193083517)
@@ -0,0 +1,17 @@
+package com.maternalcare.demo.model;
+
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "test_table")
+public class TestEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+
+    // getters & setters
+}
